src/routes/batchTransfers.ts

```diff
@@ -70,38 +118,74 @@ const findUnconfirmedBatchTransfer = (person, changeRequestId) => {
     `Could not find a batch transfer for the given change request id: '${changeRequestId}'`
   );
 
-  const { transfers } = person.unconfirmedBatchTransfers[index];
+  const { id, transfers, description, transfer_type } =
+    person.unconfirmedBatchTransfers[index];
 
   return {
     index,
+    id,
     transfers,
+    description,
+    transferType: transfer_type,
   };
 };
 
 export const confirmBatchTransfer = async (person, changeRequestId) => {
-  const { transfers, index } = findUnconfirmedBatchTransfer(
-    person,
-    changeRequestId
-  );
+  const { id, transfers, description, transferType, index } =
+    findUnconfirmedBatchTransfer(person, changeRequestId);
 
   person.unconfirmedBatchTransfers.splice(index, 1);
 
   const acceptedTransfers = transfers.map((transfer) => ({
```

Comment:
in theory we should accept up to exhausting account balance, but for simplicity maybe thats fine